packages/twenty-front/src/modules/ui/layout/top-bar/TopBar.tsx

```diff
@@ -7,14 +7,18 @@ type TopBarProps = {
   rightComponent?: ReactNode;
   bottomComponent?: ReactNode;
   displayBottomBorder?: boolean;
+  removeSpacing?: boolean;
```

Comment:
@rathodvinod5 Have you identified places where we want to keep the left padding?
We should enforce consistent layout accross the app and we should find a good design rule. If the API has too man parameters that are not justified, it will be harder to maintain

Reply:
Yes, currently there are only 2 places from where the component being called.

Reply:
can we remove the padding everywhere? does it make sense?